homework1/styles.css

```diff
@@ -0,0 +1,652 @@
+@font-face {
+  font-family: Baloo-Regular;
+  src: url(./fonts/Baloo-Regular.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Regular;
+  src: url(./fonts/Baloo2-Regular.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Medium;
+  src: url(./fonts/Baloo2-Medium.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Regular;
+  src: url(./fonts/Baloo2-Regular.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Medium;
+  src: url(./fonts/Baloo2-Medium.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-SemiBold;
+  src: url(./fonts/Baloo2-SemiBold.ttf);
+}
+
+@font-face {
+  font-family: Baloo2-Bold;
+  src: url(./fonts/Baloo2-Bold.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  -webkit-tap-highlight-color: transparent;
+}
+body {
+  font-family: "Baloo2-Regular", sans-serif;
+  line-height: 1.6;
+  margin: 0 auto;
+  min-height: 100vh;
+}
+
+.header--content,
+.main-nav {
+  display: flex;
+}
+
+.header--content {
+  flex-direction: row;
```

Comment:
`flex-direction: row;` is a default state for flex. Don't need to declarate it. Only when u need to rewrite styles.